### PR TITLE
Text content, grammar, and spelling

### PR DESCRIPTION
**Pull request by FFincannon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,73 @@
...
 
-You can set up adb debugging over WiFi: https://developer.android.com/studio/command-line/adb#wireless
+## Additional Resources
 
-## Additional resources <a name="additional_resources"></a>
-1. Yubico - [Developers website](https://developers.yubico.com)
-2. Yubico - [Online Demo](https://demo.yubico.com)
-![]()
+* Yubico - [Developers' website](https://developers.yubico.com)
+* Yubico - [Online Demo](https://demo.yubico.com) for OTP and U2F
```
Comment (dainnilsson) [inline README.md:67]: Replace U2F with WebAuthn.
The Playground part of the demo supports both YubiOTP and OATH, not sure if we want to leave it at just "OTP" as it is now, or if we want to specify.

File: management/README.md [hunk 2]
```diff
@@ -36,36 +40,49 @@ yubikitVersion=1.0.0-beta05
...
+2. Turn off/on YubiKey applets and transports
+
 
 #### Programming OTP slots
-Things to consider before running this demo: 
-* By default the slot ONE is programmed with YubiOTP secret. Overriding first slot means loosing this secret.
-* YubiOTP codes that were generated on key can be read using "Yubico OTP demo" pivot. Refer to the [OTP module](../otp/README.md) to learn more.
+Before running this demo, be aware that:
+* By default slot ONE is programmed with the YubiOTP secret. Overwriting this first slot means losing this secret.
+* YubiOTP codes that were generated on the YubiKey can be read using the *Yubico OTP demo* pivot. Refer to the [OTP Module](../otp/README.md) to learn more.
```
Comment (dainnilsson) [inline management/README.md:77]: I'm not familiar with this usage of the word "pivot", and I can't really find any similar usage by Googling, so I'm not sure where it comes from. It seems to refer to the different "parts" of the demo app, maybe we can just use _demo_ for this? "... using the *Yubico OTP* demo", and so on?

File: README.md [hunk 3]
```diff
@@ -1,38 +1,73 @@
-# YubiKit for Android
-**This is a pre-release version of YubiKit. Some of the specs and APIs may change in future releases so use this documentation and the library for prototyping and not for a public release. If you are an early adopter please provide detailed feedback about the design and issues you may find when using YubiKit.**
+# Yubico Mobile Android SDK (YubiKit)
 
-**YubiKit**  is an Android library provided Yubico to interact with YubiKeys. The library supports NFC-enabled and USB YubiKeys.
+The **YubiKit** is an Android library provided by Yubico to interact with YubiKeys on Android devices. The library supports NFC-enabled and USB YubiKeys.
```
[the remark this answers is not in the document]
Comment (FFincannon) [reply README.md:3]: Removed the definite article throughout

File: README.md [hunk 4]
```diff
@@ -1,38 +1,73 @@
...
 
-Please note, **YubiKit** consumes data from the token and translates to the application for further processing. **YubiKit** performs data validation for the purposes of creating valid responses to the application, however content/logical validation and security decisions are outside the scope of the **YubiKit** and must be handled by the application.
+The library includes a demo application implemented in Kotlin, the [YubiKit Demo App](./yubikit-android/tree/master/YubikitDemo), which provides a complete example of integrating and using all the features of the library in an Android project.
+
+Changes to this library are documented in this [changelog](./yubikit-android/blob/master/Changelog.md).
+
+
+## About
+
+The YubiKit requires a physical key to test its features. Before running the included [demo application](./yubikit-android/tree/master/YubikitDemo) or integrating the YubiKit into your own app, you need a YubiKey or a Security Key by Yubico to test functionality.
```
[the remark this answers is not in the document]
Comment (FFincannon) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -1,38 +1,73 @@
...
 
 **YubiKit** is multi-module library with following components:
 
-[Yubikit](./yubikit/README.md) provides an functionality to detect YubiKey plugged in or tapped over NFC,
+[Yubikit](./yubikit/README.md) provides functionality to detect whether a YubiKey is plugged in or is connected over NFC.
+
+[OATH](./oath/README.md) allows applications such as an authenticator app to store OATH TOTP and HOTP secrets on a YubiKey and generate one-time passwords.
```
[the remark this answers is not in the document]
Comment (FFincannon) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (FFincannon) [reply README.md:69]: I specified "for YubiOTP, OATH and WebAuthn"

File: management/README.md (hunk 2 above)
Comment (FFincannon) [reply management/README.md:77]: I know "pivot" only as something the company does when we change direction, and in the context of Excel - so I'm happy to get rid of it.

File: README.md [hunk 6]
```diff
@@ -1,38 +1,79 @@
...
+# Yubico Mobile Android SDK (YubiKit)
 
-**YubiKit**  is an Android library provided Yubico to interact with YubiKeys. The library supports NFC-enabled and USB YubiKeys.
+**YubiKit** is an Android library provided by Yubico to enable interaction between YubiKeys and Android devices. The library supports NFC-enabled and USB YubiKeys.
 
-Please note, **YubiKit** consumes data from the token and translates to the application for further processing. **YubiKit** performs data validation for the purposes of creating valid responses to the application, however content/logical validation and security decisions are outside the scope of the **YubiKit** and must be handled by the application.
+The library includes a demo application implemented in the Kotlin playground, the [YubiKit Demo App](./yubikit-android/tree/master/YubikitDemo), which provides a complete example of integrating and using all the features of the library in an Android project.
```
Comment (dainnilsson) [inline README.md:5]: What is the Kotlin playground?

Comment (FFincannon) [reply README.md]: "Kotlin Playground is an online sandbox to explore Kotlin
programming language. Browse code samples directly in the browser": https://play.kotlinlang.org/#eyJ2ZXJzaW9uIjoiMS4zLjcyIiwicGxhdGZvcm0iOiJqYXZhIiwiYXJncyI6IiIsImpzQ29kZSI6IiIsIm5vbmVNYXJrZXJzIjp0cnVlLCJ0aGVtZSI6ImlkZWEiLCJjb2RlIjoiLyoqXG4gKiBZb3UgY2FuIGVkaXQsIHJ1biwgYW5kIHNoYXJlIHRoaXMgY29kZS4gXG4gKiBwbGF5LmtvdGxpbmxhbmcub3JnIFxuICovXG5cbmZ1biBtYWluKCkge1xuICAgIHByaW50bG4oXCJIZWxsbywgd29ybGQhISFcIilcbn0ifQ==

File: oath/README.md [hunk 7]
```diff
@@ -52,23 +52,23 @@ First, create an `OathApplication` to select OATH applet on YubiKey.
...
+In addition, the `OathApplication` provides an interface for setting/validating a password on credential storage, calculating all credentials, and resetting the OATH application to its default state. For the complete list of methods look at the [`OathApplication` class documentation](src/main/java/com/yubico/yubikit/oath/OathApplication.java).  
 
-The **OATH** module also provides a class for defining an OATH `Credential`. Use its convenience initializer `Credential.parseUri` to parse the credential parameters from Uri of [Key Uri Format](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
+The **OATH** module also provides a class for defining an OATH `Credential`. Use its convenient initializer `Credential.parseUri` to parse the credential parameters from the Uri of the [Key Uri Format](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
```
Comment (dainnilsson) [inline oath/README.md:64]: Change "convenience initializer" to "factory method" which is a more commonly used term.

**Comment by dainnilsson** [comment]
One thought I had: In the YubikitDemo directory we have a QuickStart.md file, whereas in other modules we have README.md files instead. It would be more consistent if we use README.md everywhere, and that has the benefit of rendering as the main page in Github when you click on the directory (compare https://github.com/Yubico/yubikit-android/tree/master/YubikitDemo with for example https://github.com/Yubico/yubikit-android/tree/master/yubikit). Should we rename the QuickStart.md file?

**Pull request merged**